labs/lab-1-html-and-css/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html> 
+	<head>
+		<title>Chirpus.inc</title>
+		<link rel="stylesheet" type="text/css" media=" (min-width:701px) " href="style.css">
+		<link rel="stylesheet" media=" (max-width: 700px)" href="mobilestyle.css">
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:400" rel="stylesheet">
+	</head>
+	<body>
+
+		<nav>
+			<ul class="nav">
```

Comment:
I'm not a fan of the classname `nav` -- seeing `.nav {` in the css always looks like a mistake. Maybe try `navbar-links` or something like that.